conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
```

Comment:
To me the new API we are aiming for will be relatively simple (consisting of a few classes / methods only). It's a *public* API we are designing, not the internal API this uses.
As such, I'd expect this to live directly one level under `conda`, rather than `conda.core`.
(I have to admit that I don't fully understand the rationale of the `conda` package (sub-)structure, but as most of it is not supposed to be public API, I wouldn't spend much time questioning it. But for the public API this is different...)